Description: Fix type mismatch between qreal and double on ARM EABI.
Author: Jakub Wilk <jwilk@debian.org>
Bug-Debian: http://bugs.debian.org/644250
Forwarded: no
Last-Update: 2011-11-13

--- a/src/fminfodisplay.cpp
+++ b/src/fminfodisplay.cpp
@@ -151,7 +151,7 @@
 				GlyphToSVGHelper gtsh ( gpi->path(), tf );
 				svg += gtsh.getSVGPath() + "\n";
 				horOffset += gpi->data(GLYPH_DATA_HADVANCE).toDouble() * scaleFactor;
-				maxHeight = qMax ( gtsh.getRect().height(), maxHeight );
+				maxHeight = qMax ( (double)gtsh.getRect().height(), maxHeight );
 				tf.translate( gpi->data(GLYPH_DATA_HADVANCE).toDouble()  * scaleFactor,0 );
 				delete gpi;
 			}
